source/manual/covid-19-services.html.md

```diff
@@ -4,416 +4,115 @@ title: COVID-19 Services
-- [GitHub](https://github.com/alphagov/govuk-coronavirus-find-support)
-
-## Architecture
-
-All applications have a similar architecture of:
-
-- **Framework:** Ruby on Rails
-- **Database:** AWS RDS (business volunteering, find support) and DynamoDB (vulnerable people)
-- **Hosting:** PaaS
-- **CDN:** AWS CloudFront
-- **CI:** GitHub Actions (branches and PRs)
-- **CD:** Concourse (master builds and production deploys)
-- **Logging:** Sentry and Logit
-- **Email**: GOV.UK Notify
-- **Queuing**: Sidekiq (business volunteering only)
-
-### Application structure
-
-Each application is a standard Rails application with:
-
-- question pages, each has their own controller, view and route
-- a check your answers page (except find support form)
-- a confirmation ("Thank you") page
-- ineligible pages (vulnerable people form only)
-- a privacy page
-
-To run one of the applications locally, see the README in the GitHub repo.
-
-## Session and data storage
-
-### Session storage
-
-While the user is filling out the form, we use session storage to
-store the user's data.
-
-For the vulnerable people form and find support form, we store session data for
-4 hours in an encrypted cookie, persisted in the browser and sent back to the
-server for every question.  Cookies have a limit of 4KB, so this approach could
-cause errors if the user submits large inputs.
-
-For the business volunteering form, we store the following for 4 hours:
-
-- the session id in an encrypted cookie
-- the session data in Redis
-
-### Data storage
-
-For all but the find support form, when the user submits the form on
-the "Check your answers" page, the application writes user data to the
-database.
-
-For the find support form, user responses are written to the database
-on submission of the final question. There is no "Check your answers"
-page.
-
-The vulnerable people form application only has permission to write
-items to the database. For security and privacy reasons, there's no
-way to read or change already-submitted user data.
-
-Developers should not have access to the production database for
-the vulnerable people form application.
-
-The business volunteering form and find support form applications can both read
-and write to their databases as the security and privacy requirements are lower.
-
-Developers may treat data in the business volunteering form and find support
-form to the same standards as any other GOV.UK personal data store, and are able
-to access it for legitimate development duties.
-
-## Hosting
+**The service has now been migrated and is now a smart answer.**
 
-### PaaS
+A backup of the production database was taken prior to the service
+being removed. It is available in the `govuk-production-database-backups`
+AWS S3 bucket under `coronavirus-find-support/production.sql.gzip`
+with 365 day retention period (expiry date: Fri, 08 Oct 2021 00:00:00 GMT)
+as per our privacy policy.
```

Comment:
Again, I think this information would be more readable if it was presented in a bullet list:

```
A backup of the production database was taken prior to the service
being removed. It is available at:

- AWS S3 bucket - `govuk-production-database-backups`
- Key - `coronavirus-find-support/production.sql.gzip`
- 365 day retention period (as per our privacy policy)
- expiry date - Fri, 08 Oct 2021 00:00:00 GMT
```

Reply:
Thanks @reggieb - fixed 👍